exercises/affine-cipher/src/test/java/AffineCipherTest.java

```diff
@@ -0,0 +1,130 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public class AffineCipherTest {
+
+    private AffineCipher affineCipher;
+
+    @Before
+    public void genAffineCipfer() {
+        affineCipher = new AffineCipher();
+    }
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void testEncodeYes() {
+        assertEquals("xbt", affineCipher.encode("yes", 5, 7)
+        );
```

Comment:
I still think this should be on the previous line? So that it becomes: 
```
assertEquals("xbt", affineCipher.encode("yes", 5, 7));
```
What do you think? 🙂

Reply:
yes, seems I didn't exactly understand your previous comment. Of course I will correct it! 😃